### ISSUE TITLE
Update favicon for more aesthetically pleasing option.

### ISSUE THREAD

**Issue by ImMattDavison** [twitter/opensource-website#144]
**Is your feature request related to a problem? Please describe.**
The issue is not related to an problem, the idea is simply aesthetic but if the favicon.ico file had round edges it may be more appealing/aesthetically pleasing.

**Describe the solution you'd like**
I would like to update the favicon.ico file to a more appealing less sharp-edged image with round edges instead. alternatives can be found at https://about.twitter.com/en/who-we-are/brand-toolkit

**Describe alternatives you've considered**
While the current favicon serves it's purpose, it could be replaced with a round edged square or circle with the blue background and white twitter icon

**Additional context**
I am happy to contribute this change myself, this is the current favicon which I find slightly unpleasing to the eye: 
![image](https://user-images.githubusercontent.com/80921257/185965615-26c9eef0-29cb-40d8-9c80-8d773b858b28.png)
I intend to replace it with either of these images: 
![Twitter social icons - circle - blue](https://user-images.githubusercontent.com/80921257/185966042-a513e288-99d6-443d-8b28-4dbcd5f015a7.png)
![Twitter social icons - rounded square - blue](https://user-images.githubusercontent.com/80921257/185966160-4f02de64-25ce-4887-81a0-3ec30207644b.png)

**Comment by ArslanYM** [comment]
Can i work on this please?

**Comment by manan1979** [comment]
hi @ImMattDavison assign this to me

**Comment by wrunkel** [comment]
Hey @ImMattDavison,

i have some time right now and could replace the favicon.

You could just assign the issue to me.

**Comment by 0xSaksham** [comment]
@ImMattDavison This issue is easy, should I start working on it? if yes please assign it to me.

**Comment by duhhbzz** [comment]
I agree @ImMattDavison, your proposed images are clean & smooth. Kudos
